tests/boot/boot_to_desktop.pm

```diff
@@ -29,8 +29,8 @@ sub run {
     # Add additional 60 seconds if the test suite is migration as reboot from
     # pre-migration system may take an additional time. Booting of encrypted disk
     # needs additional time too.
-    $timeout += 60 if get_var('PATCH') || get_var('ONLINE_MIGRATION');
-    $timeout += 60 if get_var('ENCRYPT');
+    $timeout += 120 if get_var('PATCH') || get_var('ONLINE_MIGRATION');
+    $timeout += 60  if get_var('ENCRYPT');
```

Comment:
If this code is for generic one, i suggest add extra line instead of direct change the timeout.
$timeout += 60  if is_pvm;